aten/src/ATen/cpu/vec256/vec256_bfloat16.h

```diff
@@ -428,30 +427,28 @@ Vec256<BFloat16> inline operator^(const Vec256<BFloat16>& a, const Vec256<BFloat
   return _mm256_xor_si256(a, b);
 }
 
-const Vec256<BFloat16> Vec256<BFloat16>::ones(1.0f);
-
 Vec256<BFloat16> Vec256<BFloat16>::eq(const Vec256<BFloat16>& other) const {
-  return (*this == other) & Vec256<BFloat16>::ones;
```

Comment:
I meant adding a line like this here:

    static const Vec256<BFloat16> ones(1.0f);

Reply:
This is still going to introduce an atomic guard for initialization